devtoolsNG/basic/index.html

```diff
@@ -0,0 +1,36 @@
+<body>
+  <div id="log"></div> 
+  <input type="button" onclick="clearEntries()" value="Clear">
+  <script>
+
+    const websocket = new WebSocket('ws://localhost:12345');

```

Comment:
Potentially trickier: parameterizing / using constants for the port everywhere would be worthwhile for flexibility and programmatic/config-based configuration.